### ISSUE TITLE
Instructor Help : Change 'To' to 'Evaluee/Recipient' in rubric question example

### ISSUE THREAD

**Issue by sukanta-27** [TEAMMATES/teammates#8571]
While I was working on #8176, I did a search for the term `Evaluee` in the `InstructorHelp` page and changed the words to suggested `Evaluee/Recipient`. However I forgot to do a search for `To` which I assumed was replaced to `Evaluee` in #7900. But it seems one occurrence of this word got overlooked and that is in the `Example 10` under `Rubric question`. Change the following `To`->`Evaluee/Recipient` 
to make it consistent with the rest of the UI.

**Current :**
![screenshot 392](https://user-images.githubusercontent.com/10681755/36887915-269eadf4-1deb-11e8-83de-be8b5f9b2ec6.png)

**How it should be:**
![screenshot 391](https://user-images.githubusercontent.com/10681755/36887931-3d9cdbc0-1deb-11e8-8681-ca4abf16f28f.png)

Additionally cross check for any other such occurances.

**Comment by sukanta-27** [comment]
This is a `d.FirstTimers` issue.

**Comment by wkurniawan07** [comment]
While difficulty-wise it is, I'm not sure if this is something that should be fixed "only when we feel like it". @damithc ?

**Comment by damithc** [comment]
> While difficulty-wise it is, I'm not sure if this is something that should be fixed "only when we feel like it". @damithc ?

Yes, not very urgent but it's good to make the documentation consistent with the UI. Thanks for the report @sukanta-27

**Comment by Pratyum** [comment]
I am new to this and would like to work on this issue Please let me know if it is possible

**Comment by sukanta-27** [comment]
@Pratyum There is already an open PR for this issue, you can work on a different `d.FirstTimers` issue if you like.
